src/containers/shared/components/Transaction/AMM/AMMSharedSimple.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleProps } from '../types'
+import { localizeBalance, localizeNumber } from '../../../utils'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+
+export const TRADE_FEE_TOTAL = 1000
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const {
+    amount,
+    amount2,
+    lpTokens,
+    accountID,
+    effectivePrice,
+    fee,
+    tradingFee,
+    bidMin,
+    authAccounts,
+  } = data.instructions
+  const value =
+    amount?.amount && amount.amount !== fee ? (
+      <Amount value={amount} />
+    ) : undefined
+  const value2 =
+    amount2?.amount && amount2.amount !== fee ? (
+      <Amount value={amount2} />
+    ) : undefined
+  const lpTokenFormatted = lpTokens
+    ? localizeNumber(lpTokens, 'en-US', {
+        minimumFractionDigits: 0,
+        maximumFractionDigits: 2,
+      })
+    : undefined
+  const bid = bidMin ? localizeBalance(bidMin) : undefined
+  const tf = tradingFee
```

Comment:
There should be a shared function to calculate trading fee.

Reply:
done - moved it to utils.